stdlib/3/os/__init__.pyi

```diff
@@ -607,9 +614,14 @@ def popen(cmd: str, mode: str = ..., buffering: int = ...) -> _wrap_close: ...
 def spawnl(mode: int, file: _PathType, arg0: Union[bytes, Text], *args: Union[bytes, Text]) -> int: ...
 def spawnle(mode: int, file: _PathType, arg0: Union[bytes, Text],
             *args: Any) -> int: ...  # Imprecise sig
-def spawnv(mode: int, file: _PathType, args: List[Union[bytes, Text]]) -> int: ...
-def spawnve(mode: int, file: _PathType, args: List[Union[bytes, Text]],
-            env: _ExecEnv) -> int: ...
+if sys.platform != "win32":
+    def spawnv(mode: int, file: _PathType, args: List[Union[bytes, Text]]) -> int: ...
```

Comment:
Not related to your PR, but the args argument can also be a tuple. It also allows the file and args to be ints:
```
In [63]: os.spawnv(mode=42, file=43, args=(1, 2))
Out[63]: 127
```
But I'm not sure whether that 's meaningful.